obspy/io/sac/sactrace.py

```diff
@@ -1105,12 +1102,15 @@ def read(cls, source, headonly=False, ascii=False, byteorder=None,
 
         .. rubric:: Example
 
-        >>> sac = SACTrace.read(filename, headonly=True) # doctest: +SKIP
-        >>> try: # doctest: +SKIP
-                sac.validate('data_hdrs') # doctest: +SKIP
-            except SacInvalidContentError: # doctest: +SKIP
-                sac = SACTrace.read(filename, headonly=False) # doctest: +SKIP
-                sac.validate('data_hdrs') # doctest: +SKIP
+        >>> from obspy.core.util import get_example_file
+        >>> from obspy.io.sac.util import SacInvalidContentError
+        >>> file_ = get_example_file("test.sac")
+        >>> sac = SACTrace.read(file_, headonly=True)
+        >>> try:
+        ...     sac.validate('data_hdrs')
+        ... except (ValueError, SacInvalidContentError):
+        ...     sac = SACTrace.read(file_, headonly=False)
+        ...     sac.validate('data_hdrs')
```

Comment:
Dunno.. maybe just fixing the header (i.e. setting with actual mean) in the test file would make sense? (Unless that makes othere tests fail.. ;-)